source/index.html

```diff
@@ -2,11 +2,164 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+    <header class="header">
+      <div class="container">
+        <a href="#" class="logo">
+          <img src="" alt="Логотип" class="logo__img">
+        </a>
+        <nav class="menu menu--open">
+          <button class="menu__button" type="button" name="menu-toggle"><span class="visually-hidden">Открыть меню</span></button>
+          <ul class="menu__list">
+            <li class="menu__item">Главная</li>
+            <li class="menu__item">Каталог продукции</li>
+            <li class="menu__item">Подбор программы</li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+
+      <section class="promo">
+        <div class="container">
+          <h1 class="promo__title">Функциональное питание для котов</h1>
+          <p class="promo__text">Занялся собой? Займись котом!</p>
+          <a class="promo__button" href="#">Подобрать программу</a>
+          <img src="" alt="Банка корма" class="promo__img">
+        </div>
+      </section>
+
+      <section class="catalogs">
+        <div class="container">
+          <ul class="catalogs__list">
+            <li class="catalogs__item">
+              <div class="catalogs__item-box">
+                <div class="catalogs__item-header">
+                  <img class="catalogs__item-icon" src="" alt="Иконка каталога">
+                  <h2 class="catalogs__item-title">похудение</h2>
```

Comment:
Лучше текст вставлять в обычном виде, как в предложениях, первая буква - заглавная
```suggestion
                  <h2 class="catalogs__item-title">Похудение</h2>
```